src/all/misskon/src/eu/kanade/tachiyomi/extension/all/misskon/MissKon.kt

```diff
@@ -76,46 +77,37 @@ class MissKon() : SimpleParsedHttpSource() {
         return SManga.create().apply {
             title = postInnerEl.select(".post-title").text()
             genre = postInnerEl.select(".post-tag > a").joinToString { it.text() }
+            status = SManga.COMPLETED
+            update_strategy = UpdateStrategy.ONLY_FETCH_ONCE
         }
     }
 
-    override fun chapterListSelector() = "html"
-
-    override fun chapterFromElement(element: Element): SChapter {
-        val dateStr = element.selectFirst(".entry img")?.absUrl("data-src")
+    override fun chapterFromElement(element: Element) = throw UnsupportedOperationException()
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val doc = response.asJsoup()
+        val dateUploadStr = doc.selectFirst(".entry img")?.absUrl("data-src")
             ?.let { url ->
                 FULL_DATE_REGEX.find(url)?.groupValues?.get(1)
                     ?: YEAR_MONTH_REGEX.find(url)?.groupValues?.get(1)?.let { "$it/01" }
             }
-
-        return SChapter.create().apply {
-            chapter_number = 0F
-            setUrlWithoutDomain(element.selectFirst("link[rel=canonical]")!!.absUrl("href"))
-            name = "Gallery"
-            date_upload = FULL_DATE_FORMAT.tryParse(dateStr)
+        val dateUpload = FULL_DATE_FORMAT.tryParse(dateUploadStr)
+        val maxPage = doc.select("div.page-link:first-of-type a.post-page-numbers").last()?.text()?.toInt() ?: 1
+        val basePageUrl = response.request.url.toString()
+        return (maxPage downTo 1).map { page ->
+            SChapter.create().apply {
+                chapter_number = 0F
```

Comment:
don't set chapter number to 0, this will cause the app to filter out or mark all chapters as read when these settings are enabled

Reply:
seems like you removed it in the wrong place